reference/curl/functions/curl-upkeep.xml

```diff
@@ -0,0 +1,101 @@
+<?xml version="1.0" encoding="utf-8"?>
+<refentry xml:id="function.curl_upkeep" xmlns="http://docbook.org/ns/docbook" xmlns:xlink="http://www.w3.org/1999/xlink">
+ <refnamediv>
+  <refname>curl_upkeep</refname>
+  <refpurpose>Führt alle Prüfungen zur Aufrechterhaltung einer Verbindung durch</refpurpose>
```

Comment:
That's kind of inaccurate to me (though also an appropriate translation from English). 

Does the function really perform checks to maintain a connection? Or doesn't the function rather try to keep the connection open?

Reply:
Yes, I think so, too. Maybe this should be changed accordingly in [doc-en](https://github.com/php/doc-en/blob/master/reference/curl/functions/curl-upkeep.xml).

Reply:
Thank you. I will have a closer look at the function and then fix the whole thing in the English documentation if necessary.